app/sdil/config/RoutesWiring.scala

```diff
@@ -25,7 +25,7 @@ import sdil.actions.{AuthorisedAction, FormAction, RegisteredAction, VariationAc
 import sdil.connectors.{ContactFrontendConnector, GaConnector, SoftDrinksIndustryLevyConnector, TestConnector}
 import sdil.controllers._
 import sdil.controllers.test.TestingController
-import sdil.controllers.variation.{BusinessDetailsController, VariationsController}
+import sdil.controllers.variation.{BusinessDetailsController, ContactDetailsVariationController, VariationsController}
```

Comment:
Might as well make this a wildcard import if there are going to be more added especially

Reply:
Done